projects/netgrif-components-core/src/assets/i18n/de.json

```diff
@@ -306,6 +306,12 @@
             "sign": "Registrieren",
             "enterPass": "Ihre Kennwort eingeben"
         },
+        "changePassword": {
+            "oldPassword": "Geben Sie Ihr altes Passwort ein",
+            "enterNewPass": "Geben Sie Ihr neues Passwort ein",
+            "repeatNewPass": "Wiederholen Sie Ihr neues Passwort",
```

Comment:
Texts are not consistent with `forgottenPass` texts.

Why do we define duplicate texts anyway?